hphp/runtime/vm/jit/vasm-simplify.cpp

```diff
@@ -431,6 +431,21 @@ bool simplify(Env& env, const copyargs& inst, Vlabel b, size_t i) {
   });
 }
 
+bool simplify(Env& env, const loadb& inst, Vlabel b, size_t i) {
+  return if_inst<Vinstr::movzbl> (env, b, i+1, [&] (const movzbl& cm) {
+    // loadb; movzbl; -> loadzbl;
+    if (!(arch() == Arch::ARM &&
+          env.use_counts[inst.d] == 1 &&
+          env.use_counts[cm.s] == 1 &&
```

Comment:
I'm referring to the specific line I've attached this comment to.

If `inst.d == cm.s`, then obviously `env.use_counts[inst.d] == 1` will imply that `env.use_counts[cm.s] == 1`.